sql/statesql/schema/schema.sql

```diff
@@ -0,0 +1,151 @@
+PRAGMA user_version = 18;
```

Comment:
How do we ensure that this schema is in line with the migrations? Is there a test that fails if I add a migration and forget to update this here?